Dockerfile

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1
+
+# Install the base requirements for the app.
+# This stage is to support development.
+FROM ubuntu:latest AS base
+
+# install prerequisits
+######################
+
+RUN \
+  echo "install packages" && \
+  apt-get update && \
+  apt-get install -y \
+	curl \
+    jq \
+    unzip \
+    python3 \
+    python3-pip \
+    python3-venv
+
+RUN apt-get install -y sqlcipher libsqlcipher-dev
```

Comment:
any reason not to install these in the install line above?

Reply:
I was having some issues where it didn't install sqlcipher correctly when on the same line, but only on some devices 🤷‍♂️. Separating it seemed to fix the issue